build.zig

```diff
@@ -78,26 +101,40 @@ fn formatToString(f: Format) []const u8 {
     return @tagName(f);
 }
 
-fn getShdcLazyPath(dep_shdc: *Build.Dependency) !Build.LazyPath {
-    const intel = builtin.cpu.arch.isX86();
-    const opt_sub_path: ?[]const u8 = switch (builtin.os.tag) {
-        .windows => "bin/win32/sokol-shdc.exe",
-        .linux => if (intel) "bin/linux/sokol-shdc" else "bin/linux_arm64/sokol-shdc",
-        .macos => if (intel) "bin/osx/sokol-shdc" else "bin/osx_arm64/sokol-shdc",
-        else => null,
-    };
-    if (opt_sub_path) |sub_path| {
-        return dep_shdc.path(sub_path);
-    } else {
-        return error.ShdcUnsupportedPlatform;
+pub fn getShdcSubPath() error{ShdcUnsupportedPlatform}![]const u8 {
```

Comment:
This might be a useful export as before accidentally finding `compile` is exported here I was recreating this fn to use with `b.addSystemCommand` directly. This still might be useful in some cases, i.e currently `shdc_dir` is a relative path someone might want to do an absolute path or other things.